pkg/components/aws-ebs-csi-driver/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package awsebscsidriver - The golang plumbing for the aws-ebs-csi-driver
+// component
```

Comment:
```suggestion
// Package awsebscsidriver contains code for the aws-ebs-csi-driver component.
```

- Make the doc string follow the Go [conventions](https://blog.golang.org/godoc) for package documentation.
- Explain what the package does.
- Add a period at the end.